check-approved-requests.py

```diff
@@ -38,7 +39,8 @@ def parse_user_row(cells):
     user_info = {'user_name': email,
                  'user_email': email,
                  'first_name': cells[4],
-                 'last_name': cells[5]}
+                 'last_name': cells[5],
+                 'project': cells[14]}
```

Comment:
It's not always going to be `cells[14]`... sometimes it will be `cells[17]` if they are being added to an existing project.

There are already if statements navigating that, so I suggest adding `project` to the dictionary in there.